test/extended/olm/olmv1-singleownnamespace.go

```diff
@@ -0,0 +1,65 @@
+package operators
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"time"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[sig-olmv1][OCPFeatureGate:NewOLMOwnSingleNamespace][Skipped:Disconnected] OLMv1 operator single/own namespace support", func() {
```

Comment:
We need to come up with 3 more tests to satisfy component readiness and the bar for GA. Maybe some negative tests?

Reply:
Hey Joe, I wasn't aware of that.
Do you happen to have a doc link that describes all feature GA requirements?